contracts/superclasses/Posts.sol

```diff
@@ -262,7 +268,10 @@ contract Posts is KetlGuarded {
       participantsMap[feedId][postId][sender] = true;
     }
     // Increment comments count
-    parentPost.numberOfComments++;
+    posts[feedId][postId].numberOfComments++;
+    if (replyTo > 0) {
+      comments[feedId][postId][replyTo - 1].numberOfComments++;
```

Comment:
Do we need to update the comment's number of comments? I don't think we display it anywhere in the UI

Reply:
Probably not, let's ask Sergey, maybe he needs it